docs/status.md

```diff
@@ -27,21 +27,21 @@ one of the following tracking labels.
 
 ## Status
 
-| StableHLO Op (114)       | Specification (25) | Verification (104) | Type Inference (82) | Pretty Printing (77) | Interpreter (11) |
+| StableHLO Op (114)       | Specification (25) | Verification (114) | Type Inference (83) | Pretty Printing (77) | Interpreter (11) |
```

Comment:
What does "Verification (114)" mean in this context? I would expect that it would be a number of ops, where we've done everything we can at the moment (i.e. "yes" or "yes*"), but it looks like it also includes ops that need revisiting? Same question for type inference.

Reply:
I clarify this the new description. Please see accordingly. Numbers are removed previous PRs from other contributors.